main.tf

```diff
@@ -86,6 +96,16 @@ resource "cloudflare_record" "staging_cname" {
   allow_overwrite = true
 }
 
+resource "cloudflare_record" "staging_xchain_cname" {
```

Comment:
why do we need to add another resource here? let's just call it staging and use the same one as before

Reply:
shouldn't this replace the blockscout ui?

Reply:
no we have a separate api server address that the blockscout ui calls, this url is for the api server.

Reply:
I don't want to make it `<something>.staging.explorer.omni.network` because `staging.explorer.omni.network` is itself an app so might cause issues so I'll just say `staging-xapi.explorer.omni.network` does that work?

Reply:
yeah